opentrons-ai-client/src/molecules/ChatDisplay/index.tsx

```diff
@@ -327,7 +327,11 @@ export function ChatDisplay({ chat, chatId }: ChatDisplayProps): JSX.Element {
           </Markdown>
         )}
         {protocol_content != null && (
-          <span>
+          <StyledText
+            fontSize={TYPOGRAPHY.fontSize20}
+            lineHeight={TYPOGRAPHY.lineHeight24}
+            css="white-space: pre-wrap;"
```

Comment:
as i commented, we use `desktopStyle` to set font-size, line-height.
components/src/ui-style-constants/typography.ts

```suggestion
           desktopStyle={smallBodyTextRegular}
           whiteSpace={WHITE_SPACE_PRE_WRAP}
```

Reply:
If I set like so, font size does not match with remaining. 

```
          desktopStyle="bodyDefaultRegular"
          whiteSpace={WHITE_SPACE_PRE_WRAP}
```
<img width="1390" alt="image" src="https://github.com/user-attachments/assets/7476c631-1d62-42cb-8cf3-6afcea2a4d2c" />


if we look at figma https://www.figma.com/design/viYBfEgz3TEnqHqfgH9B7S/Opentrons-AI?node-id=3549-4377&t=pWhGceDE0PYzWKRW-0
typography is 20px

<img width="1329" alt="image" src="https://github.com/user-attachments/assets/d8142d27-867a-4d9c-ad7f-f3cef5ddb572" />


That is why I set to 20px.

Reply:
I also tried `oddStyle="smallBodyTextRegular"` it is not working.

Reply:
oddStyle is for ODD screen size

Reply:
Did you check `comupted` font-size, line-height, and font-weight when using ` desktopStyle="bodyDefaultRegular"`